doc/api/stream.md

```diff
@@ -1261,16 +1261,27 @@ added: v11.4.0
 Is `true` if it is safe to call [`readable.read()`][stream-read], which means
 the stream has not been destroyed or emitted `'error'` or `'end'`.
 
+##### `readable.readableAborted`
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* {boolean}
+
+Returns whether the stream was destroyed before all data has been read.
```

Comment:
I'm unsure whether this should be:

`Returns whether the stream was destroyed before all data has been read.`

or

`Returns whether the stream was destroyed before `'end'` was emitted.`